src/backend/app/organization/organization_crud.py

```diff
@@ -44,15 +44,12 @@ def generate_slug(text: str) -> str:
 
 
 async def get_organisation_by_name(db: Session, name: str):
-
-    # Construct the SQL query with the case-insensitive search
-    query = f"SELECT * FROM organisations WHERE LOWER(name) LIKE LOWER('%{name}%') LIMIT 1"
-
-    # Execute the query and retrieve the result
-    result = db.execute(query)
-
-    # Fetch the first row of the result
-    db_organisation = result.fetchone()
+    # Use SQLAlchemy's query-building capabilities
```

Comment:
Try to avoid using SQLAlchemy, as we will remove it in the future.
The raw SQL equivalent is probably best 👍